.ci/jobs/packaging.yml

```diff
@@ -28,9 +28,6 @@
               ignore-tags-older-than: -1
               ignore-tags-newer-than: 30
           - named-branches:
```

Comment:
probably unrequired

Reply:
removed in 5a47bd21893fa53edee283931b47f8941c88b86c